src/proc/zero-order.cpp

```diff
@@ -103,7 +103,7 @@ namespace librealsense
             return val == 0;
         }), values_ir.end());
 
-        if (values_rtd.size() == 0 || values_rtd.size() == 0)
+        if ((values_rtd.size() == 0) || (values_ir.size() == 0))
```

Comment:
Why the extra ()?
Using .empty() instead of the comparison to 0 would be more readable.

Reply:
Agreed on using .empty() instead and dropping the "extra ()".

I updated my forked branch. What else do I need to do to update the PR?